### DISCUSSION TITLE
How to work on Git Hub

### DISCUSSION THREAD

**Discussion by qprizing** [community/community#22894] [New to GitHub]
<p>Hello, There is a site [edited to remove link] now we want to refine it. We found a freelancer, but he wants to work for Git Labs and asks for the code to be posted there. He also said to put the site on Git Hub for work. For the first time I work with this service and I want to clarify:</p>
<ol>
<li>How to do this?</li>
<li>Can hosting help this?</li>
<li>Is it possible to restrict access to the code to a circle of persons?</li>
<li>Will this affect the operation of the site?</li>
<li>Is it possible to work here not only with PHP, but also with <a href="http://asp.net" rel="nofollow noopener">asp.net</a> core, as we plan to translate this platform?</li>
</ol>
<p>Sorry for my bad english</p>

**Comment by lee-dohm** [comment, marked as answer]
<ol>
<li>Take a look at our help documentation, specifically the <a href="https://help.github.com/categories/bootcamp/" rel="nofollow noopener">bootcamp section</a> for instructions on your first steps to hosting software on GitHub</li>
<li>Software code hosting can help with having a place to share code amongst a team, yes</li>
<li>Yes, it is. See the <a href="https://help.github.com/articles/access-permissions-on-github/" rel="nofollow noopener">help documentation on access permissions</a>
</li>
<li>Storing a copy of the source code of the site on a code host will not affect the operation of the site in any way, no</li>
<li>You can use whatever language, technology, or framework you wish when storing your source code here</li>
</ol>
<p>I hope that helps!</p>

**Reply by Elyadazamy** [reply]
Thanks fot help I never don't know that every things chicked from others

**Comment by Elyadazamy** [comment]
Sorry
